packages/xdl/src/DevSession.ts

```diff
@@ -1,6 +1,8 @@
 import { ExpoConfig } from '@expo/config-types';
 import os from 'os';
+import { URLSearchParams } from 'url';
 
+import { ProjectSettings } from '.';
```

Comment:
I think these need to be imported from './internal' or something like that, not entirely sure why.